core/classes/Endpoints.php

```diff
@@ -0,0 +1,38 @@
+<?php 
+/*
+ *	Made by Aberdeener
+ *
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0-pr8
+ *
+ *  License: MIT
+ *
+ *  Endpoints class
+ */
+class Endpoints {
+
+    private 
+        $_endpoints = array();
+
+    public function __construct() {}
+
+    public function getAll() {
+        return $this->_endpoints;
+    }
+
+    public function add(EndpointBase $endpoint) {
```

Comment:
Can we remove the EndpointBase type from here too?

Reply:
As per our conversation on Discord, leaving this.